docs/deployment/breaking-changes/6.7-client-python-metrics.md

```diff
@@ -0,0 +1,23 @@
+# Renaming of metrics name in the python client
+
+!!! info ""
+
+    * **Changed in**: `OpenCTI 6.7`
```

Comment:
```suggestion
    * **Changed in**: `OpenCTI 6.8`
```

Reply:
it will be packed in the next major relaese

Reply:
Thank you!